Features/Team 1/Semeniuk/Feature with OOP project(class Person)/TestProjects/Program.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+namespace TestProjects
+{
+    public class Person
+    {
+
+        public int age { get; set; }
+        public string name { get; set; }
+        public string sex { get; set; }
+        public long number { get; set; }
+
+        public Person(int age, string name, string sex, long number)
+        {
+            this.age = age;
+            this.name = name;
+            this.sex = sex;
+            this.number = number;
+        }
+        public void ChangeName(string name)
+        {
+            this.name = name;
+        }
+        public void ChangeSex(string sex)
+        {
+            this.sex = sex;
+        }
+        public void ChangeAge(int age)
+        {
+            this.age = age;
+        }
+        public void ChangeNumber(long number)
+        {
+            this.number = number;
+        }
+        public void PrintPerson()
+        {
+            Console.WriteLine($"Age is {age}, Name is {name}, Sex is {sex}, Phone number is {number}");
+        }
+
+    }
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Input data about person(Age,name,sex,number)");
+            int age = int.Parse(Console.ReadLine());
+            string name = Console.ReadLine();
+            string sex = Console.ReadLine();
+            long number = Convert.ToInt64(Console.ReadLine());
+            Person myObj = new Person(age, name, sex, number);
+            Console.WriteLine("If u want to change something press y");
+            while (Console.ReadLine()=="y")
+            {
+                Console.WriteLine("Choose what to change age(1),name(2),sex(3),numer(4)");
+                int n = int.Parse(Console.ReadLine());
+                switch (n)
+                {
+                    case 1:
+                        Console.WriteLine("Input new count of age");
+                        myObj.ChangeAge(Convert.ToInt32(Console.ReadLine()));
+                        break;
+                    case 2:
+                        Console.WriteLine("Input new count of Name");
+                        myObj.ChangeName(Console.ReadLine());
+                        break;
+                    case 3:
+                        Console.WriteLine("Input new count of Sex");
+                        myObj.ChangeSex(Console.ReadLine());
+                        break;
+                    case 4:
+                        Console.WriteLine("Input new count of Phone number");
+                        myObj.ChangeNumber(Convert.ToInt64(Console.ReadLine()));
+                        break;
+                    default:
```

Comment:
Do we need the empty default block?

Reply:
If somebody gonna press, not 1-4, then default activate

Reply:
It will finish this switch-case cycle code even without an empty default block (I've checked ;) ).
We need a default option only to execute some code for the default case.

Reply:
Changed, for a message to users who wrote wrong number

Reply:
I think it is right to add a default case

Reply:
If it still wrong I`ll remove it